spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/LocalDeployerSupportTests.java

```diff
@@ -93,6 +93,32 @@ public void testCalcServerPort() throws MalformedURLException {
 		assertThat(portToUse, not(9292));
 	}
 
+	@Test
+	public void testShutdownPropertyConfiguresRequestFactory() throws Exception {
+		LocalDeployerProperties properties = new LocalDeployerProperties();
+		properties.setShutdownTimeout(1);
+		AbstractLocalDeployerSupport abstractLocalDeployerSupport = new AbstractLocalDeployerSupport(properties) {};
+		Object restTemplate = TestUtils.readField("restTemplate", abstractLocalDeployerSupport);
```

Comment:
Think we can use `ReflectionTestutils` in the spring-test package instead of our own util class.

```
		Object restTemplate = ReflectionTestUtils.getField(abstractLocalDeployerSupport, "restTemplate");
		Object requestFactory = ReflectionTestUtils.getField(restTemplate, "requestFactory");
		Object connectTimeout = ReflectionTestUtils.getField(requestFactory, "connectTimeout");
		Object readTimeout = ReflectionTestUtils.getField(requestFactory, "readTimeout");```

will check other usages.